app/javascript/store/modules/profile/actions.js

```diff
@@ -25,6 +30,19 @@ export default {
     dispatch(COMPUTE_RECOMMENDATIONS, { teamId, githubUserLogin });
   },
 
+  [PROCESS_NEW_ORGANIZATION](
+    { commit }, {organizationId, githubUserLogin, teams, selectedTeamId}) {
```

Comment:
A space is required after '{'                            object-curly-spacing<br>'githubUserLogin' is defined but never used              no-unused-vars<br>'selectedTeamId' is defined but never used               no-unused-vars<br>A space is required before '}'                           object-curly-spacing